packages/react-component-library/src/components/TabSet/TabSet.tsx

```diff
@@ -1,44 +1,81 @@
 import React, { useState, Children } from 'react'
+import classNames from 'classnames'
 
 import { TabProps, TabContent, TabItem } from '.'
+import { ScrollButton } from './ScrollButton'
+import { useScrollableTabSet } from './useScrollableTabSet'
+import { SCROLL_DIRECTION } from './constants'
 
 interface TabSetProps {
   className?: string
-  children?: React.ReactElement<TabProps>[]
-  onChangeCallback?: (id: number, title: string) => void
+  children: React.ReactElement<TabProps>[]
+  onChangeCallback?: (id: number) => void
+  scrollable?: boolean
 }
 
 export const TabSet: React.FC<TabSetProps> = ({
   className = '',
   children,
   onChangeCallback,
+  scrollable,
 }) => {
   const [activeTab, setActiveTab] = useState(0)
+  const { updateCurrentScrollToTab, tabsRef, itemsRef } = useScrollableTabSet(
+    children
+  )
 
-  function handleClick(index: number, title: string) {
+  function handleClick(index: number) {
     setActiveTab(index)
 
     if (onChangeCallback) {
-      onChangeCallback(activeTab, title)
+      onChangeCallback(index)
```

Comment:
I know this was already here before and it's annoyingly a breaking change but maybe we could lose the word 'callback' and it could become `onChange`.

Reply:
Maybe there is a task here to carry out ahead of the v2.0.0 release - we could make our interfaces more consistent across the entire library.